src/resolvers/__tests__/removeById-test.ts

```diff
@@ -167,7 +168,7 @@ describe('removeById() ->', () => {
       expect(exist?.n).toBe(user.name);
     });
 
-    it('should call `beforeQuery` method with non-executed `query` as arg', async () => {
+    it.only('should call `beforeQuery` method with non-executed `query` as arg', async () => {
```

Comment:
Could you please remove `.only`? It skips a bunch of tests.

Reply:
Ugh... yea! removed 👍